tests/defaults_list/data/experiment/override_with_global_default2.yaml

```diff
@@ -0,0 +1,2 @@
+defaults:
+  - override /group1@_global_: file2
```

Comment:
It's because the file the experiment is overriding, `group_default_at_global`, is defined as 👍🏼 
```yaml
defaults:
  - group1@_global_: file1
```